src/objecttracker.cpp

```diff
@@ -37,7 +37,7 @@ NAN_METHOD(getObjectHistogram) {
 
 	HeapProfiler *heapProfiler = isolate->GetHeapProfiler();
 
-#if NODE_VERSION_AT_LEAST(4, 0, 0) // > v0.11+
+#if NODE_VERSION_AT_LEAST(4, 0, 0) // > v4.00+
```

Comment:
unrelated to this PR, but https://github.com/bnoordhuis/node-heapdump/blob/master/src/heapdump.cc#L69-L78 (which is copied into this repo) has a TakeHeapSnapshot(), maybe it can just be called from heapsnapshot/heapsnapshot.cc?

Reply:
opened a new issue https://github.com/RuntimeTools/appmetrics/issues/579